docs/source/examples.rst

```diff
@@ -7,6 +7,132 @@ example documents in the
 with instructions on how to download additional documents that are too large to store in the
 repo.
 
+
+##########################################
+Sentiment Analysis Labeling in LabelStudio
+##########################################
+
+The following workflow will show how to format and upload the risk section from an SEC filing
+to LabelStudio for a sentiment analysis labeling task. In addition to the ``unstructured``
+library, this examples assumes you have the
+`SEC pipeline <https://github.com/Unstructured-IO/pipeline-sec-filings>`_ repo installed and
+on your Python path, as well `LabelStudio <https://labelstud.io/guide/#Quick-start>`_ installed
+and running locally. First, we'll import dependencies and pull in the risk section of an
+SEC filing:
+
+.. code:: python
+
+    import json
+
+    from prepline_sec_filings.sec_document import SECDocument, SECSection
+    from prepline_sec_filings.fetch import get_form_by_ticker
+
+    from unstructured.staging.label_studio import (
+        stage_for_label_studio,
+        LabelStudioAnnotation,
+        LabelStudioPrediction,
+        LabelStudioResult,
```

Comment:
We'll need to bump the `unstructured` version in `pipeline-sec-filings` for this code to work (not a problem for this code tho).

Reply:
(Assuming they are getting `unstructured` from the `pipeline-sec-filings` environment, I guess that's not a given.)

Reply:
Updated this to hard code the risk narrative elements, per Crag's comment.